test/contrib/hdfs_test.py

```diff
@@ -386,6 +386,12 @@ def create_target(self, format=None):
             target.remove(skip_trash=True)
         return target
 
+    def create_flag_target(self, format=None):
```

Comment:
the method name here is confusing. You aren't actually creating the flag target. Just the class object needed to later create the flag

Reply:
I'm not sure what you mean here, this method does create a `HdfsFlagTarget`, and the helper method is modeled off of the similar method called `create_target` line 385. 

But in any case I don't think the helper method is necessary (especially if only used in that once test) and can be removed and the FlagTarget created in the test